docs/quickstart.md

```diff
@@ -0,0 +1,760 @@
+    You can read more about local testing under following link: [SAM local testing](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-local-start-api.html)
+!!! warning
+    **Powertools Tracer** requires X-RAY service to work. It means that you won't be able to see traces locally. Deploy example to your AWS account instead.
+#### Remote test
+You may also deploy your application into AWS Account by issuing following command.
+```bash
+> sam build && sam deploy --guided
+...
+CloudFormation outputs from deployed stack
+------------------------------------------------------------------------------------------------------------------------------------------
+Outputs                                                                                                                                  
+------------------------------------------------------------------------------------------------------------------------------------------
+Key                 HelloWorldFunctionIamRole                                                                                            
+Description         Implicit IAM Role created for Hello World function                                                                   
+Value               arn:aws:iam::123456789012:role/sam-app-HelloWorldFunctionRole-1T2W3H9LZHGGV                                          
+
+Key                 HelloWorldApi                                                                                                        
+Description         API Gateway endpoint URL for Prod stage for Hello World function                                                     
+Value               https://1234567890.execute-api.eu-central-1.amazonaws.com/Prod/hello/                                                
+
+Key                 HelloWorldFunction                                                                                                   
+Description         Hello World Lambda Function ARN                                                                                      
+Value               arn:aws:lambda:eu-central-1:123456789012:function:sam-app-HelloWorldFunction-dOcfAtYoEiGo                           
+------------------------------------------------------------------------------------------------------------------------------------------
+Successfully created/updated stack - sam-app in eu-central-1
+```
+This command will build package and deploy it to your AWS Account. Make sure you configure AWS credentials beforehand. In the outputs section you find API Url you can issue request against.
+!!! Info
+    To learn more about SAM deployment mechanism follow this [link](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-deploy.html).
+
+## API Gateway router
+Let's say you have been asked to create another method that respond with caller name under `/hello/<name>` URL path.
```

Comment:
Is that an echo method? If yes, maybe we can first describe the problem (return input) and then the implementation (use path variable in url. Suggestion:

Let's say we want to have a method that outputs a string that we send back, like an echo. We would need to create an API with an URL path `/hello/<message>`, where the message is the input from the user.

Reply:
Added something like this:
```Let's say we want to have another method that behaves as an echo. It takes user input (username) and output it to the caller. We would need to create an API with an URL path /hello/<name>, where the name string is the input from the user.
```